runtime-config.md

```diff
@@ -15,25 +15,25 @@ Additional filesystems can be declared as "mounts", specified in the *mounts* ar
         "type": "proc",
         "source": "proc",
         "destination": "/proc",
-        "options": ""
+        "options": {}
     },
     {
         "type": "tmpfs",
         "source": "tmpfs",
         "destination": "/dev",
-        "options": "nosuid,strictatime,mode=755,size=65536k"
+        "options": {"nosuid":true,"strictatime":true,"mode":"755","size":"65536k"}
     },
     {
         "type": "devpts",
         "source": "devpts",
         "destination": "/dev/pts",
-        "options": "nosuid,noexec,newinstance,ptmxmode=0666,mode=0620,gid=5"
+        "options": {"nosuid":true,"noexec":true,"newinstance":true,"ptmxmode":"0666","mode":"0620","gid":5}
     },
     {
         "type": "bind",
         "source": "/volumes/testing",
         "destination": "/data",
-        "options": "rbind,rw"
+        "options": {"rbind":true,"rw":true}
```

Comment:
It doesn't expect to be true, it expects to be `rw`. The pattern is, if there is a string on the right hand side(RHS) then the syntax is `item=string` and if it is `true` then the syntax is `item` without an equals sign. That's why I suggested using `nil` rather than `true`.